pkgs/development/python-modules/autobahn/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, buildPythonPackage, fetchurl, isPy3k,
+  unittest2, mock, pytest, trollius, pytest-asyncio,
+  six, twisted, txaio
+}:
+buildPythonPackage rec {
+  name = "${pname}-${version}";
+  pname = "autobahn";
+  version = "0.18.2";
+
+  src = fetchurl {
+    url = "mirror://pypi/a/${pname}/${name}.tar.gz";
+    sha256 = "1alp71plqnrak5nm2vn9mmkxayjb081c1kihqwf60wdpvv0w7y14";
+  };
+
+  buildInputs = [ unittest2 mock pytest trollius pytest-asyncio ];
+  propagatedBuildInputs = [ six twisted txaio ];
+
+  disabled = !isPy3k;
```

Comment:
You are right, this is missplaced. This should be  in `pytest-asyncio` which is python3 only. But the problem will remain, tests cant pass for `autobahn` for `python2`. I have opened a PR to fix it and build autobahn for `python2` while skiping tests (#25460).